torchgeo/datasets/spacenet.py

```diff
@@ -554,6 +554,272 @@ def __init__(
+    and Khartoum.
+
+    Collection features:
+
+    +------------+---------------------+------------+---------------------------+
+    |    AOI     | Area (km\ :sup:`2`\)| # Images   | # Road Network Labels (km)|
+    +============+=====================+============+===========================+
+    | Vegas      |    216              |   854      |         3685              |
+    +------------+---------------------+------------+---------------------------+
+    | Paris      |    1030             |   257      |         425               |
+    +------------+---------------------+------------+---------------------------+
+    | Shanghai   |    1000             |   1028     |         3537              |
+    +------------+---------------------+------------+---------------------------+
+    | Khartoum   |    765              |   283      |         1030              |
+    +------------+---------------------+------------+---------------------------+
+
+    Imagery features:
+
+    .. list-table::
+        :widths: 10 10 10 10 10
+        :header-rows: 1
+        :stub-columns: 1
+
+        *   -
+            - PAN
+            - MS
+            - PS-MS
+            - PS-RGB
+        *   - GSD (m)
+            - 0.31
+            - 1.24
+            - 0.30
+            - 0.30
+        *   - Chip size (px)
+            - 1300 x 1300
+            - 325 x 325
+            - 1300 x 1300
+            - 1300 x 1300
+
+    Dataset format:
+
+    * Imagery - Worldview-3 GeoTIFFs
+
+        * PAN.tif (Panchromatic)
+        * MS.tif (Multispectral)
+        * PS-MS (Pansharpened Multispectral)
+        * PS-RGB (Pansharpened RGB)
+
+    * Labels - GeoJSON
+
+        * labels.geojson
+
+    If you use this dataset in your research, please cite the following paper:
+
+    * https://arxiv.org/abs/1807.01232
+
+    .. note::
+
+       This dataset requires the following additional library to be installed:
+
+       * `radiant-mlhub <https://pypi.org/project/radiant-mlhub/>`_ to download the
+         imagery and labels from the Radiant Earth MLHub
+
+    .. versionadded:: 0.3
+    """
+
+    dataset_id = "spacenet3"
+    collection_md5_dict = {
+        "sn3_AOI_2_Vegas": "8ce7e6abffb8849eb88885035f061ee8",
+        "sn3_AOI_3_Paris": "90b9ebd64cd83dc8d3d4773f45050d8f",
+        "sn3_AOI_4_Shanghai": "3ea291df34548962dfba8b5ed37d700c",
+        "sn3_AOI_5_Khartoum": "b8d549ac9a6d7456c0f7a8e6de23d9f9",
+    }
+
+    imagery = {
+        "MS": "MS.tif",
+        "PAN": "PAN.tif",
+        "PS-MS": "PS-MS.tif",
+        "PS-RGB": "PS-RGB.tif",
+    }
+    chip_size = {
+        "MS": (325, 325),
+        "PAN": (1300, 1300),
+        "PS-MS": (1300, 1300),
+        "PS-RGB": (1300, 1300),
+    }
+    label_glob = "labels.geojson"
+
+    def __init__(
+        self,
+        root: str,
+        image: str = "PS-RGB",
+        speed_mask: Optional[bool] = False,
+        collections: List[str] = [],
```

Comment:
Would it be better to default to all collections?

Reply:
It actually already does this in the base class. Refer [here](https://github.com/microsoft/torchgeo/blob/b2e178f1fef028af8d0f0e96bc45c34ee0030c83/torchgeo/datasets/spacenet.py#L102)

Reply:
It might help to explain this in the class docstring so users know that it doesn't actually default to the empty list.